content/collections/account-management/en/user-roles-permissions.md

```diff
@@ -41,6 +41,7 @@ Recommended role for third parties, if you don't want them creating content that
 * Share content they create with others
 * View Guides and Surveys
 * Create and view [Heatmaps](/docs/session-replay/heatmaps)
+* Receive notification alerts
```

Comment:
My bad on this. You dont need to be viewer to receive the notification alerts. I mean you need viewer to see https://app.amplitude.com/analytics/amplitude/settings/experiment/subscriptions to see the row in the table for that project